app/routes/models/calculation-bps.js

```diff
@@ -0,0 +1,168 @@
+const bands = require('../../calculation/bands')
+const schemeYears = require('../../calculation/scheme-years')
+const toCurrencyString = require('../../utils/to-currency-string')
+
+function ViewModel (value, calculations) {
+  this.model = {
+    confirmation: createSummary(value),
+    paymentBand: createTableDefinition(calculations, { property: 'rate', text: '', caption: 'Progressive reductions by scheme year', formatType: 'percentage', showOverall: false }),
+    reduction: createTableDefinition(calculations, { property: 'reduction', text: 'Total progressive reduction:', caption: 'Progressive reductions applied to your claim by scheme year', formatType: 'currency', showOverall: true }),
+    paymentSummary: createPaymentSummary(calculations, { caption: 'Estimated BPS payments 2021 to 2023' }),
+    backLink: createBackLink()
```

Comment:
There is no reference to this property in the calling model, hence it is redundant and should be removed.

Reply:
redundant reference removed